packages/core/test/scxml.test.ts

```diff
@@ -358,14 +358,15 @@ async function runW3TestToCompletion(machine: MachineNode): Promise<void> {
         nextState = state;
       })
       .onDone(() => {
-        if (nextState.value === 'pass') {
+        // Add 'final' for test230.txml which does not have a 'pass' state
```

Comment:
eh, it's classified as a manual test - not sure rly why as this one would be totally possible to be rewritten as an automated one (it would grow a little bit due to added messages passing and context assignments but whatever)

🤷‍♂️